doc/lenet.txt

```diff
@@ -400,6 +400,14 @@ tensors. These are then flattened to a 2D matrix of rasterized feature maps,
 to be compatible with our previous MLP implementation.
 
 
+.. note::
+    Please note that the convolution operation in majority of of the recent
+    models and almost all the library codes available is slightly different
+    than the original LeNet model. In modern conv2d, for each feature map
+    The result sums all the convolved input feature maps with the corresponding
+    kernel. As opposed to some of the earlier works that used to select only
+    some of the input feature maps.
+
```

Comment:
"in majority" => "in the majority"
"of of" => "of"
"library codes" => "implementations" ?
"is slightly different than" => "differs slightly from"
Starting at "In modern conv2d" until the end, it's not really obvious to me what you mean. This would probably benefit from being rephrased. There are a few typos and errors too, but I'll wait since this will probably be rewritten anyway.